domains/measurements/Measurements.API/Tests/Fixtures/TestServerHelpers/ForwardLoggingProvider.cs

```diff
@@ -0,0 +1,74 @@
+#region Copyright notice and license
+// Copyright 2019 The gRPC Authors
```

Comment:
What is this?

Reply:
Taken from the wallet, which got it from somewhere else. It basically says it's been copy pasted